CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contribute
+
+## Install
+
+```console
+yarn install
```

Comment:
We already use `pnpm`. Essentially there are no reasons not to.

Reply:
Contributing guidelines is a huge topic for Meaningful, so it's developed separately organization-wide: https://linear.app/meaningful/team/SYM/active